packages/api-providers/src/polkadot/vanchor-deposit.ts

```diff
@@ -190,6 +190,8 @@ export class PolkadotVAnchorDeposit extends VAnchorDeposit<WebbPolkadot, Deposit
         extAmount: BigNumber.from(publicAmount),
         encryptedOutput1: u8aToHex(comEnc1),
         encryptedOutput2: u8aToHex(comEnc2),
+        refund: '0',
+        token: '1',
```

Comment:
The `extData` value should be consistent with the value in the `ProvingManagerSetupInput`. I would suggest a change to both for something like:

extData - wrappableAssetRaw.toString() ?? '0'
provingInput - hexToU8a(wrappableAssetRaw.toString()) ?? Uint8Array.from([0])